src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/partials/SidebarBanner.tsx

```diff
@@ -21,28 +22,54 @@ export const SidebarBanner: FC = () => {
     ({ key }) => key !== 'title',
   );
 
-  const actionTypeNotificationTitle =
+  const actionTypeNotification =
     useCreateActionTypeNotification(selectedAction);
   const actionTypeNofiticationHref =
     useCreateActionTypeNotificationHref(selectedAction);
 
-  if (actionTypeNotificationTitle) {
+  const getProperty = (obj, key, defaultValue) => {
```

Comment:
this looks like a useful util! maybe we can already put it to `utils/objects` already :)